osu.Game/Screens/OnlinePlay/Playlists/ClosePlaylistDialog.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using osu.Game.Online.Rooms;
+using osu.Game.Overlays.Dialog;
+
+namespace osu.Game.Screens.OnlinePlay.Playlists
+{
+    public partial class ClosePlaylistDialog : DeletionDialog
+    {
+        public ClosePlaylistDialog(Room room, Action closeAction)
+        {
+            HeaderText = "Are you sure you want to close the following playlist:";
```

Comment:
I hesitantly mentioned this in my other comment, but we could just expose this to the user as "deleting" and not require custom terminology for closing. I think to the user they expect to "delete" a playlist, not "close" it.. but I can swing both ways.

Reply:
Am neither here or there, fine with either personally.